impl_core/src/main/java/io/opencensus/implcore/stats/MeasureMapImpl.java

```diff
@@ -61,6 +72,13 @@ public void record() {
 
   @Override
   public void record(TagContext tags) {
+    if (hasUnsupportedValues) {
+      // drop all the recorded values
+      builder.clear();
```

Comment:
this is a change that we should not add it right now. I think record should not reset the values.

Reply:
done